pymc3/stats.py

```diff
@@ -581,8 +584,8 @@ def add_warns(*args):
         warnings.filterwarnings('always')
 
         ics = []
-        for c, (t, m) in enumerate(zip(traces, models)):
-            ics.append((c, ic_func(t, m, pointwise=True)))
+        for m, t in model_dict.items():
+            ics.append((m.name, ic_func(t, m, pointwise=True)))
 
```

Comment:
add the line `c += 1`

Reply:
add indentation, sorry for not mentioning before but you should increase `c` inside the `for-loop`